R/QC_images.R

```diff
@@ -164,10 +164,17 @@ QC_images = function(sample_meta, raw_counts, annotated_counts, normalized_count
   run_below= F
   if(run_below) {
     # Determine which seq cols are present.
-    # sample_meta sequencing columns
-    sm_seq_cols= c('flowcell_name', 'flowcell_lane', 'index_1', 'index_2')
-    # raw_counts sequencing columns
-    rc_seq_cols= c('flowcell_name', 'flowcell_lane', 'index_1', 'index_2')
+    if("index_2" %in% colnames(sample_meta)){
```

Comment:
should the seq colulmns be plural flowcell_names and flowcell_lanes?

Reply:
This section of code was a WiP from the last pull request. It doesn't run because its wrapped in an if statement. The code has been fixed in the other pull request that hasn't been pushed yet